### PR TITLE
Added game configuration files for Doom Builder 2.0

### PR DESCRIPTION
**Pull request by akaAgar**
Contains all sectors, linedefs and things from EDGE Classic 1.35, properly sorted by category.

Tested with DoomBuilder 2.0, should work will all its forks. Could be useful for level designers.

### CODE REVIEW

**Comment by LoboEire** [comment]
We need to decide what do do with the various Doombuilder configs we've got floating around. We've also got DB.x ones which have been passed around on Discord but not included "offcially" anywhere. I imagine they are compatible with DB2 also.

At present we've been focussing more on UDB, which seems to be the most popular map editor currently. We've had UDB ones for quite some time: the DiD ones are actually included with the program. 
We haven't added the newest (Did + UDMF) publicly yet but if you're interested you can check this out: https://edge-classic.github.io/tutorials/index.htm

File: extras/DoomBuilder configurations/Edge_Classic.cfg [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+// Door making
+makedoortrack = "DOORTRAK";
+makedooraction = 1;			// See linedeftypes
+
+// Generalized actions
+generalizedlinedefs = true;
+generalizedsectors = true;
+
+// Texture loading options
+mixtexturesflats = false;
```
Comment (dashodanger) [inline extras/DoomBuilder configurations/Edge_Classic.cfg:54]: I'm not too familiar with DB2, but if this refers to being able to use flats as textures and vice versa, EC is capable of doing so

**Comment by LoboEire** [comment]
I have added the configs for UDB and DoomBuilder.x to our repo:
https://github.com/LoboEire/EDGE-classic/commit/c0dad96e79a752383c4473499f78419540450aab

I'm fairly sure the db.x ones would also work with db2: if you are using db2, can you try them?

**Comment by dashodanger** [comment]
I tested our existing DB.X configs with DB2, and they indeed do not work with it; will pull this in and then do some pushes later to rearrange it so that it sits with the other config folders

**Pull request merged**